<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutOfTime</class>
 <widget class="QDialog" name="OutOfTime">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="SkippedTurnMessage">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Out of time! </string>
   </property>
  </widget>
  <widget class="QLabel" name="NextTurnMessage">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;p's&gt; turn in </string>
   </property>
  </widget>
  <widget class="QLabel" name="Countdown">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
